.github/workflows/nightly.yml

```diff
@@ -363,12 +452,22 @@ jobs:
           export SYMFONY_DEPRECATIONS_HELPER=max[total]=999
           X=0
           for component in $(find src/Symfony -mindepth 2 -type f -name phpunit.xml.dist -printf '%h\n'); do
-            php ./phpunit $component --exclude-group tty,benchmark,intl-data,transient --exclude-group skip || EXIT_CODE=$?
-            if [ $EXIT_CODE -gt 128 ]; then
+            php $GITHUB_WORKSPACE/.github/patch.php ./phpunit $component --exclude-group tty,benchmark,intl-data,transient --exclude-group skip || EXIT_CODE=$?
+            if [ ${EXIT_CODE:-0} -gt 128 ]; then
               X=1;
             fi
           done
           exit $X
+      - name: Test PHPSeclib
+        if: always()
+        run: |
+          git clone https://github.com/phpseclib/phpseclib --branch=master
+          cd phpseclib
+          git checkout 259bd9f1e8af11726ed74acf527c2c046549061b
```

Comment:
Is there a reason why this is pinned to a specific commit? I think we should avoid having to bump library versions.

Reply:
I would like to avoid the tests breaking due to changes in the libraries...

Reply:
Nevermind, removed the static tag :)